src/components/common/Label/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { StyledLabel } from './styled';
+
+const Label = ({ text, isRequired }) => (
```

Comment:
@BrownMark I opted not to use container/presentational for this component as it is so basic with no internal state. But can refactor if you think the other approach is better.

Reply:
Oh that's fine really. The big thing is convention.

If there's an established convention we need to make sure to adhear to it.